MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -10,23 +10,41 @@ import SwiftUI
 
 final class MapViewModel: ObservableObject {
 
-    @Published var selectedFarmerPlace: FarmerPlace?
+    @Published var selectedFarmer: Farmer?
+
+    @Published var allFarmers: [Farmer] = []
 
     @Published var userPosition: MapCameraPosition = .userLocation(fallback: .automatic)
 
-    let allFarmerPlaces = FarmerPlace.all
+    init() {
+        readJsonFarmers()
+    }
 
     func onViewAppear() {
         requestUserAuthorization()
     }
 
+    private func readJsonFarmers() {
```

Comment:
Je t'avais mis qq chose a ce propos, le naming n'est pas bon. Le service tu comptes le faire dans une autre PR ? Aucun soucis si oui mais si c'est oui, n'hesite pas a ajouter un todo (`// TODO: to move into a service`). Ca aide aussi les lecteurs a comprendre tes intentions, la je sais pas ce que tu comptes faire. Et en plus ca te met un reminder pour toi